src/js/components/PageHeaderActionsMenu.js

```diff
@@ -0,0 +1,62 @@
+import {Dropdown} from 'reactjs-components';
+import React from 'react';
+
+import Icon from './Icon';
+
+const getMenuItems = (children) => {
+  return [
+    {
+      className: 'hidden',
+      html: <Icon id="ellipsis-vertical" size="mini" />,
+      id: 'trigger'
+    },
+    ...React.Children.map(children, getDropdownItemFromComponent)
+  ];
+};
+
+const handleItemSelection = (item) => {
+  if (item.clickHandler) {
+    item.clickHandler();
```

Comment:
My concern with using `onClick` is that the element which gets rendered inside the dropdown menu will have padding surrounding it. See the following screenshots:

This element has the native `onClick` handler:
![](https://cl.ly/2t013O311m1H/Screen%20Shot%202016-12-03%20at%2012.25.53%20PM.png)

This element is wrapping the child passed to `PageHeaderActionsMenu`:
![](https://cl.ly/3j281C0b0J1j/Screen%20Shot%202016-12-03%20at%2012.26.03%20PM.png)

So if the user clicks on the `span` element, the `span`'s native event handler function will get called. If they click outside the `span` element, say in the area that has padding, then the `PageHeaderActionsDropdown` will be the callee of the `onClick` function.

Additionally, if we use `onClick` we would need to proxy it in order to stop propagation. Otherwise this `onClick` method will be called twice if the user clicks on the `span` element (once for the native event handler, and again for when `handleItemSelect` calls it). This is certainly not difficult to implement, I just wanted to point that out.

Maybe `onItemSelect` could be a better prop name? Or do you think we should go with `onClick` still?